tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2133,31 +2133,54 @@ def runTest(self):
         src_port_ip = self.test_params['src_port_ip']
         src_port_vlan = self.test_params['src_port_vlan']
         src_port_mac = self.dataplane.get_mac(0, src_port_id)
-        print("dst_port_id: %d, src_port_id: %d" %
-              (dst_port_id, src_port_id), file=sys.stderr)
+        qos_remap_enable = bool(self.test_params.get('qos_remap_enable', False))
+        print("dst_port_id: %d, src_port_id: %d qos_remap_enable: %d" %
```

Comment:
How can we control this port selection? this logic will work on dual tor with pcbb only if both src and dst are vlan facing ports. similarly for T1, they have to be t2 facing ports

Reply:
For To topology, the port selection is limited to vlan ports by below logic. The ports in LAG is removed from testing ports.
https://github.com/sonic-net/sonic-mgmt/blob/f5cca011edaf0a743f3eb63889001d9332eca14d/tests/qos/qos_sai_base.py#L502-L512

For T1 topology, there is some logic error in current code, I think, because the condition in L502
https://github.com/sonic-net/sonic-mgmt/blob/f5cca011edaf0a743f3eb63889001d9332eca14d/tests/qos/qos_sai_base.py#L502
will bypass the condition in L540.
https://github.com/sonic-net/sonic-mgmt/blob/f5cca011edaf0a743f3eb63889001d9332eca14d/tests/qos/qos_sai_base.py#L540

So all ports in LAG are also removed from testing ports, which results in the test be skipped on T1-LAG topo.  This issue need to be fixed first.

Reply:
The current logic is correct. We need to exclude lag ports in mellanox platform otherwise the disable_aging script will throw some errors

Reply:
Yes, agree with you, I comment in PR #5704, but the author didn't response. The check will be removed by another PR of mine https://github.com/sonic-net/sonic-mgmt/pull/6393.